kolibri/plugins/learn/assets/src/views/content-cards/content-carousel.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+
+  <section class="content-carousel">
+
+    <header v-if="header" class="content-carousel-header">
+      <h2 v-if="header"> {{header}} </h2>
+      <sub v-if="subheader"> {{subheader}} </sub>
```

Comment:
One consideration when thinking about what tag to use is whether [the default css `display` value](https://www.w3schools.com/html/html_blocks.asp) is block-level or inline. In general, HTML is often "cleanest" when all sibling DOM nodes have the same default display: either all inline or all block. Otherwise, you need extra css to override the defaults.

So for example mixing `<h2>` with `<p>` or `<div>` is cleaner than mixing `<h2>` with `<span>` or `<sub>`.